src/JabbR-Core/Services/InMemoryRepository.cs

```diff
@@ -110,6 +101,7 @@ public void Add(Attachment attachment)
 
         public void Add(ChatRoom room)
         {
+            RoomList.Add(room);
```

Comment:
I didnt make that change. I only added the users to the light_meow room. It looks the same to me though. Not sure if that was something that happened after the merge?